### ISSUE TITLE
Runtime in comment is misleading

### ISSUE THREAD

**Issue by ketumbra** [NextronSystems/nextron-helper-scripts#5]
https://github.com/NextronSystems/nextron-helper-scripts/blob/9d3197648043b95d9676f379dbba159e83990b9e/thor-seed/thor-seed.ps1#L171
Contrary to the runtime comment on L166, L171 implies this is a quick scan when it is really the slowest of the 3 presets I think.

**Comment by redteampanda-ng** [comment]
This will be updated once https://github.com/NextronSystems/nextron-helper-scripts/tree/feature/cockpit-thor-seed is merged

**Comment by ketumbra** [comment]
oh, ok. So you'd still expect "FULL with lookback" to be much quicker than "FULL" (no global_lookback although both have lookback=14)?
I think the comment on line 195 also does not match the value on line 205.
If there's anything we can do here to clarify the distinction between these 2 presets, I think that would be really helpful.
Many Thanks

**Comment by redteampanda-ng** [comment]
Thanks for the hint. I changed the values. This was not changed after the commit 73a44630ceddc5f5cb1e8da985ec16ba40837d62. Can you check if this is clear to you now?

**Comment by ketumbra** [comment]
Erm, so 'full with lookback' is quicker because it only looks at the last 14days of any activity, whereas 'Full' looks at the complete available history (except only the last 14 days of event logs)?
Maybe 'full with lookback' could be renamed '2 week history' or 'Full (2 weeks)' and 'full' could be 'Full (no limit)' or similar?
'Lookback' implies to me that it is doing additional searching in older data and thus would be slower.
Anyway, thanks for the clarification.

**Comment by redteampanda-ng** [comment]
I agree with that the naming could be different. I will forward your suggestions internally and see if we can change the names. The script is used in other parts of our products, and customers might get confused (at first) by the new presets. So I am not sure if we will change the names anytime soon.

For the time being I added a little explanation to the two presets, which I am not sure will make it into the master branch. The reason for that is that we try to communicate with our customers (and to an extend to everyone), that they should: a) never run a script without checking what it does beforehand, and b) understand what the different THOR flags actually do. This also helps that any user of THOR understands what can and cannot be done with THOR.

We have extensive explanations in the `--fullhelp` of THOR itself, and in our documentation online (which is basically the `--fullhelp` in "pretty").
